deps/checksums/mmtk_julia

```diff
@@ -1,4 +1,2 @@
 mmtk_julia-4933fb41ba5d1d21f720e2ca1bfa5bc938b73b12.tar.gz/md5/e27debba476479953bdc23eb970f297f
 mmtk_julia-4933fb41ba5d1d21f720e2ca1bfa5bc938b73b12.tar.gz/sha512/6541bacfd16cbf1969103fea612c2f8b2011c3e38748bf82c41633c6084c2bf981bca9fb0d9f736832b0d7ef505271ba48465bcd27e8cd1fa45f7f4c5e36f246
-mmtk_julia.v0.30.6+0.x86_64-linux-gnu.tar.gz/md5/9b0c3e1ae95d82aa2184cbe889def0da
```

Comment:
Was this an accidental change? I'm not 100% sure but It looks like the Julia+MMTk build is currently broken because of it.

Reply:
I ran the refresh checksums script, which is often quite problematic. So it may have unintentionally broken that build. 

Maybe I request you to add the missing checksums back?

Reply:
No worries. Will do :)